config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get '/', to: 'welcome#index'
+
+  get '/register', to: 'register#index'
```

Comment:
We can change this to '/registration' or whatever you end up calling yours.  I just needed a placeholder